web/src/components/tasks/ElevatorRegistrationTask.test.tsx

```diff
@@ -121,6 +121,35 @@ describe("<ElevatorRegistrationTask />", () => {
       expect(screen.getByTestId("registration-2")).toBeInTheDocument();
       expect(screen.queryByTestId("registration-3")).not.toBeInTheDocument();
     });
+
+    it("shows informational messages for only relevant statuses", async () => {
```

Comment:
[pebble] - we've typically tried to separate test so that one test will assert one thing. When we are making changes to the application, it'll be a lot easier to figure out which test are failing and confirm which one should be deleted, or updated to make them passing. What are your thoughts on separating them out?